From 9e447af931264ac103169d288d675a80724a15d3 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Fri, 2 Nov 2018 09:29:31 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399pro-npu: Change tsadc clock rate
 to 650KHz

The clock frequency should be between 500KHz and 800KHz, 650KHz is
a typical value.

Change-Id: I8ee3d1e0204c6580d508029e6212e232527c644a
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi b/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
index 2dca80657642..1dab361405a5 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
@@ -279,7 +279,7 @@
 		clocks = <&cru SCLK_TSADC>, <&cru PCLK_TSADC>;
 		clock-names = "tsadc", "apb_pclk";
 		assigned-clocks = <&cru SCLK_TSADC>;
-		assigned-clock-rates = <50000>;
+		assigned-clock-rates = <650000>;
 		resets = <&cru SRST_TSADC>;
 		reset-names = "tsadc-apb";
 		#thermal-sensor-cells = <1>;
-- 
2.35.3

